### ISSUE TITLE
Content width on new install

### ISSUE THREAD

**Issue by VacantFuture** [olefredrik/FoundationPress#242]
I just setup FoundationPress on conveyor.makegroup.com. The layout is different from the http://foundationpress.olefredrik.com/kitchen-sink/ site. The content is centered on the screen and the alert boxes have a width of 970px on olefredrik.com. My install, the content is left aligned and the alert boxes have a width of 607px. 

Same thing happens when I open Google Dev Tools and select tablets (Nexus 10, Tab 7,8,10.1 or iPad 3/4). For mobile it is applying .small-12 and on the desktop is shows .large-12.

**Comment by olefredrik** [comment]
I'm not sure I understand your question. Can you please specify and possibly supply a few screenshots where you point out the errors you are experiencing on a new install?

**Comment by VacantFuture** [comment]
I hope this helps. Kitchen sink page on my install, the div with role of main is left aligned, with a width of 625px. Everything looks "squished". On your demo site, that save div has a width of 1000px and is centered on the screen.
![makengroup-kitchensink](https://cloud.githubusercontent.com/assets/2000102/6357863/4c656228-bc1c-11e4-9916-89ee2fd1632b.png)
![olefredick-kitchensink](https://cloud.githubusercontent.com/assets/2000102/6357862/4c64bb5c-bc1c-11e4-95a1-4944331ae4c8.png)

**Comment by olefredrik** [comment]
I just did a fresh install, and the Kitchen Sink template looks just fine. I suspect that this error is caused by a setting in your browser or Google Dev Tools? The row class has a max-width of 62.5rem and a width of 100%. On my 13" macbook the kitchen sink renders at 970px width. This will adapt to the viewport. Can you test in a different browser (Firefox, IE, Opera or whatever) and check if the error persists? After doing a clean install, make sure you do a ```grunt build```

**Comment by FlatspinZA** [comment]
From what I can see, somehow you are not referencing the foundation.css file, which is more than likely the problem.

**Comment by Endlyss** [comment]
@FlatspinZa
Yes, it's there. He's got it directly below the title, line 8.

**Comment by FlatspinZA** [comment]
Clearly, I am missing something:

    <head>
    <TITLE>Makegroup</TITLE>
    <meta http-equiv="Content-Type" content="text/html; charset=euc-kr">
    <script src="/js/makegroup.js" type="text/javascript"></script>
    <style type="text/css">
    A:link    { text-decoration:none; color=#BCE49A; }
    A:visited { text-decoration:none; color=#BCE49A; }
    A:hover   { text-decoration:none; color=#888888;  }
    A:hover   { text-decoration:none;  color=#9900FF;  }
    p,br,body,table,td,select,input,form,textarea,option { font-family:굴림,굴림체; font-size:8pt; }
    </style>
    </head>

**Comment by VacantFuture** [comment]
It is pointing to the wrong directory for the foundation.css

**Comment by Endlyss** [comment]
@FlatspinZA 

Apologies...between me posting "yes, it's there" and your "clearly I am missing something", something was reverted on his setup, because I am 100% sure that I saw it there, haha. Was one of the first things that looked for. I should have pasted a snippet like you did.

**Comment by VacantFuture** [comment]
Yes. I uninstalled FoundationPress to start over. Thanks for your help.

**Comment by VacantFuture** [comment]
Sorted out some server issues and did a fresh install. Unfortunately that did not fix the issue. 

This is the link to the style sheet, <link rel="stylesheet" href="http://conveyor.makengroup.com/wp-content/themes/vf-foundation/css/foundation.css"> . If you paste that into an address bar it comes up but the theme/site is not loading it?

**Comment by Aetles** [comment]
Your loading both Bootstrap and Foundation, don't think it's a good idea to use both frameworks at the same time. The `font-size: 10px;` on `html` causes the entire page to be rendered very tiny.

**Comment by VacantFuture** [comment]
Biscuits and gravy :) Everything is working correct now. 
Disabled a bootstrap shortcodes plugin i had installed some time back and forgotten about. That restored things to the proper width but it was still pushing all content to the left . A plugin that is underdevelopment and I had tested on this site earlier was pushing everything to the left. I disabled it and everything is working fine now. Thank you.
